x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -398,45 +408,8 @@ export function updateGlobalNavigation({
   updater$: Subject<AppUpdater>;
   enableExperimental: ExperimentalFeatures;
 }) {
-  const deepLinks = getDeepLinks(enableExperimental, undefined, capabilities);
```

Comment:
`getDeepLinks` already filters deepLinks using the capabilities, Cases and Alerts deepLinks will not be in the array if the read privilege is not granted, so no need to re-process it, changed to update directly.